docker-compose.yml

```diff
@@ -51,8 +51,8 @@ services:
     #   - ./tmdata:/tendermint
     entrypoint: ''
     ports:
-      - "46656"
-      - "46657"
+      - "46656:46656"
+      - "46657:46657"
```

Comment:
I though port 46657 was only used internally for RPC traffic. I don't understand why it needs to be exposed externally.

Reply:
It might be useful for debugging the system without logging in the running container.

Reply:
Maybe we should add a big disclaimer somewhere to tell people not to use Docker Compose to deploy a production node. I'll make a note to do that. Meanwhile I guess this PR can be merged.